pkg/apis/apps/types.go

```diff
@@ -100,12 +89,11 @@ const (
 	OnDeleteStatefulSetStrategyType = "OnDelete"
 )
 
-// PartitionStatefulSetStrategy contains the parameters used with the
-// PartitionStatefulSetStrategyType.
-type PartitionStatefulSetStrategy struct {
-	// Ordinal indicates the ordinal at which the StatefulSet should be
+// RollingUpdateStatefulSetStrategy is used to communicate parameter for RollingUpdateStatefulSetStrategyType.
+type RollingUpdateStatefulSetStrategy struct {
+	// Partition indicates the ordinal at which the StatefulSet should be
 	// partitioned.
-	Ordinal int32
+	Partition int32 `json:"ordinal" protobuf:"varint,1,opt,name=ordinal"`
```

Comment:
Though, this is probably better as optional. If we add a new field in rollingUpdate and set it then we will require this field to be set, too, when we may not want such a thing.